recipes-multimedia/libcamera/libcamera.bbappend

```diff
@@ -0,0 +1,5 @@
+SUMMARY = "Linux libcamera framework"
+
+PACKAGECONFIG[raspberrypi] = " -Dpipelines=raspberrypi, ,"
```

Comment:
There are some extra whitespaces you don't really need.

Reply:
FYI @agherzan 

I have updated the recipe file, as per your comments.

Reply:
Thanks. The remaining comment is abouve the subject line and dynamic layers.

Reply:
You can clean up the extra space between the commas too. If I'm not mistaken you can also omit the commas - I wouldn't bet on it though so do try it out.